src/components/Payroll/PayrollConfiguration/PayrollConfiguration.tsx

```diff
@@ -48,6 +49,16 @@ export const Root = ({ onEvent, companyId, payrollId, dictionary }: PayrollConfi
   const { mutate } = useCalculatePayrollApi({ payrollId })
   const [editedEmployeeId, setEditedEmployeeId] = useState<string | undefined>(undefined)
 
+  const { data: payScheduleData } = usePaySchedulesGet(
+    {
+      companyId,
+      payScheduleId: preparedPayroll?.payPeriod?.payScheduleUuid || '',
+    },
+    {
+      enabled: Boolean(preparedPayroll?.payPeriod?.payScheduleUuid),
```

Comment:
I would have done `!! val` here in the past, and am now considering changing to `Boolean(val)` due to it being a little less arcane